openlibrary/templates/languages/language_list.html

```diff
@@ -1,17 +1,6 @@
 $def with (classes='')
 
 <ul class="locale-options $classes">
-  <li><a href="#" lang="cs" data-lang-id="cs" title="$_('Czech')">Čeština (cs)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="de" data-lang-id="de" title="$_('German')">Deutsch (de)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="en" data-lang-id="en" title="$_('English')">English (en)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="es" data-lang-id="es" title="$_('Spanish')">Español (es)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="fr" data-lang-id="fr" title="$_('French')">Français (fr)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="hr" data-lang-id="hr" title="$_('Croatian')">Hrvatski (hr)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="it" data-lang-id="it" title="$_('Italian')">Italiano (it)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="pt" data-lang-id="pt" title="$_('Portuguese')">Português (pt)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="hi" data-lang-id="hi" title="$_('Hindi')">हिंदी (hi)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="sc" data-lang-id="sc" title="$_('Sardinian')">Sardu (sc)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="te" data-lang-id="te" title="$_('Telugu')">తెలుగు (te)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="uk" data-lang-id="uk" title="$_('Ukrainian')">Українська (uk)</a></li> $# detect-missing-i18n-skip-line
-  <li><a href="#" lang="zh" data-lang-id="zh" title="$_('Chinese')">中文 (zh)</a></li> $# detect-missing-i18n-skip-line
+  $for lang_code, lang_data in supported_langs.items():
+    <li><a href="#" lang="$lang_code" data-lang-id="$lang_code" title="$_(lang_data['localized'])">$(lang_data['native']) ($lang_code)</a></li> $# detect-missing-i18n-skip-line
```

Comment:
We don't want to use `$_` here anymore since it's already localized.
```suggestion
    <li><a href="#" lang="$lang_code" data-lang-id="$lang_code" title="$lang_data['localized']">$lang_data['native'] ($lang_code)</a></li> $# detect-missing-i18n-skip-line
```